### PR TITLE
[Bug] Fix some problem with downloading and building thirdparty

### LINKED ISSUE #5437: undefined reference to `malloc_stats_print'
OS：Ubuntu 18.04 LTS Server
Doris Version: 0.13
Error Info:
incubator-doris/thirdparty/src/rocksdb-5.14.2/db/malloc_stats.cc:51: undefined reference to `malloc_stats_print'

I just need compile Doris in real OS by source code, I have tried to download all the third parties and compile has done, but the final step has failure when "WITH_MYSQL=1 bash build.sh" directive was executed. I have been checking the third parts' version and reading the manual again, May I have something wrong with the procedures, Could you give me some help please? Our system does not permit Docker.

### PR DESCRIPTION
**Pull request by caiconghui**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have created an issue on (Fix #5437 ) and described the bug/feature there in detail
- [] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If these changes need document changes, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: thirdparty/build-thirdparty.sh [hunk 1]
```diff
@@ -472,6 +472,7 @@ build_brpc() {
     LDFLAGS="-L${TP_LIB_DIR} -static-libstdc++ -static-libgcc" \
     ${CMAKE_CMD} -G "${GENERATOR}" -v -DBUILD_SHARED_LIBS=0 -DCMAKE_INSTALL_PREFIX=$TP_INSTALL_DIR \
     -DBRPC_WITH_GLOG=ON -DWITH_GLOG=ON -DCMAKE_INCLUDE_PATH="$TP_INSTALL_DIR/include" \
+    -DCMAKE_LIBRARY_PATH="$TP_INSTALL_DIR/lib64" \
```
Comment (yangzhg) [inline thirdparty/build-thirdparty.sh:475]: now we use a soft link  lib and lib64 are same dir， so there is no need to specify DCMAKE_LIBRARY_PATH, you can clean your `installed` dir and recompile again

Comment (caiconghui) [reply thirdparty/build-thirdparty.sh:475]: I build thirdparty in ubuntu, and I find that if i don't set CMAKE_LIBRARY_PATH，it would be failed

Comment (stdpain) [reply thirdparty/build-thirdparty.sh:475]: https://github.com/apache/incubator-doris/pull/5394/files 
may be cmake found a wrong lib from system , you could reference this PR

Comment (caiconghui) [reply thirdparty/build-thirdparty.sh:475]: thank you for your reply @stdpain

**Review by yangzhg — APPROVED** [review]
LGTM

**Pull request merged**